src/main/webapp/i18n/de/global.json

```diff
@@ -32,45 +32,45 @@
                 "dropLocation": "Drop Location",
                 "dragItem": "Drag Item",
-                "participation": "Participation",
+                "participation": "Teilnahme",
                 "exerciseResult": "Ergebnis",
                 "feedback": "Feedback",
                 "submission": "Submission",
                 "modelingSubmission": "Modeling Submission",
                 "quizSubmission": "Quiz Submission",
-                "dragAndDropSubmittedAnswer": "Drag-And-Drop Submitted Answer",
+                "dragAndDropSubmittedAnswer": "Drag-And-Drop eingereichte Antwort",
                 "dragAndDropAssignment": "Drag-And-Drop Assignment",
-                "quizStatistic": "Statistics",
-                "quizPointStatistic": "Quiz Point Statistics",
-                "quizQuestionStatistic": "Question Statistics",
-                "multipleChoiceQuestionStatistic": "Multiple-Choice Question Statistics",
-                "dragAndDropQuestionStatistic": "Drag-And-Drop Question Statistics",
-                "quizStatisticCounter": "Statistic Counter",
-                "pointCounter": "Point Counter",
-                "answerCounter": "Answer Counter",
+                "quizStatistic": "Statistiken",
+                "quizPointStatistic": "Quiz Punktestatistiken",
+                "quizQuestionStatistic": "Frage Statistiken",
+                "multipleChoiceQuestionStatistic": "Multiple-Choice Frage Statistiken",
+                "dragAndDropQuestionStatistic": "Drag-And-Drop Frage Statistiken",
+                "quizStatisticCounter": "Statistikzähler",
+                "pointCounter": "Punktezähler",
+                "answerCounter": "Antwortzähler",
                 "dropLocationCounter": "Drop Location Counter",
                 "apollonDiagram": "Apollon-Diagramm",
                 "programmingSubmission": "Programming Submission",
-                "dragAndDropMapping": "Drag-And-Drop Mapping",
+                "dragAndDropMapping": "Drag-And-Drop Zuordnung",
                 "textSubmission": "Text Submission",
                 "fileUploadSubmission": "File Upload Submission",
-                "shortAnswerQuestionStatistic": "Short Answer Question Statistic",
-                "shortAnswerSpotCounter": "Short Answer Spot Counter",
-                "shortAnswerQuestion": "Short Answer Question",
-                "shortAnswerSpot": "Short Answer Spot",
-                "shortAnswerSolution": "Short Answer Solution",
-                "shortAnswerSubmittedAnswer": "Short Answer Submitted Answer",
-                "shortAnswerSubmittedText": "Short Answer Submitted Text",
-                "shortAnswerMapping": "Short Answer Mapping",
-                "programmingExerciseTestCase": "Programming Exercise Test Case",
+                "shortAnswerQuestionStatistic": "Kurzantwort Frage Statistiken",
+                "shortAnswerSpotCounter": "Kurzantwort-Spot Zähler",
+                "shortAnswerQuestion": "Kurantwort Frage",
```

Comment:
Addressed with commit 785a251c5f7265a77fad674f76ee1e6d76f75b4d and c9edf4f6f395822b135c3c13ca78b97587be351a

Reply:
Missed this occurrence. It still shows as before the suggestion.

Reply:
Sorry, my search and replace approach missed that occurrence, because it contains a typo :sweat_smile:. Fixed with commit 1e89fcd9c063d349e50675607d019336cdd6ac89.